packages/web-components/src/components/va-additional-info/va-additional-info.css

```diff
@@ -8,6 +8,10 @@ a[aria-expanded='true'] ~ #info {
   border-left-color: var(--color-primary-alt-dark);
 }
 
+#info {
+  overflow: hidden;
```

Comment:
This allows the content to become visible as the height of `#info` increases instead of all at once.